api/add_dimension.md

```diff
@@ -11,79 +11,62 @@ api:
-
-Parallel I/O can benefit in two scenarios: (a) two or more concurrent
-queries should be able to read from different disks in parallel, or
-(b) a single query should be able to use query parallelization to read
-from multiple disks in parallel.
-
-Thus, users looking for parallel I/O have two options:
-
-1.  Use a RAID setup across multiple physical disks, and expose a
-single logical disk to the hypertable (that is, via a single tablespace).
-
-1.  For each physical disk, add a separate tablespace to the
-database. Timescale allows you to actually add multiple tablespaces
-to a *single* hypertable (although under the covers, a hypertable's
-chunks are spread across the tablespaces associated with that hypertable).
-
-We recommend a RAID setup when possible, as it supports both forms of
-parallelization described above (that is, separate queries to separate
-disks, single query to multiple disks in parallel).  The multiple
-tablespace approach only supports the former. With a RAID setup,
-*no spatial partitioning is required*.
-
-That said, when using hash partitions, we recommend using 1
-hash partition per disk.
-
-Timescale does *not* benefit from a very large number of hash
-partitions (such as the number of unique items you expect in partition
-field).  A very large number of such partitions leads both to poorer
+Every distinct item in hash partitioning is hashed to one of *N* buckets. By default, 
+TimescaleDB uses flexible range intervals to manage chunk sizes. The main purpose of hash
+partitioning is to enable parallelization across multiple disks within the same time 
+interval.
+
+### Parallelizing disk I/O 
+
+You use Parallel I/O in the following scenarios:
+
+- Two or more concurrent queries should be able to read from different disks in parallel.
+- A single query should be able to use query parallelization to read from multiple disks in parallel.
+
+For the following options:
+
+- **RAID**: use a RAID setup across multiple physical disks, and expose a single logical disk to the hypertable.
+  That is, using a single tablespace.
+
+  Best practice is to use RAID when possible. This is because it supports the concurrent and single
+  scenarios and *no spatial partitioning is required*.
+
+- **Multiple tablespaces**: for each physical disk, add a separate tablespace to the database. TimescaleDB allows you to add 
+  multiple tablespaces to a *single* hypertable. However, although under the hood, a hypertable's
+  chunks are spread across the tablespaces associated with that hypertable.
+
+  Multiple tablespaces only supports concurrent queries.
+
+When using hash partitions, best practice is to have at least one hash partition per disk.
+
+set number of partitions to a multiple of number of disks. For example, the number of 
+partitions P=N*Pd where N is the number of disks and Pd is the number of partitions per 
+disk. This enables you to add more disks later and move partitions to the new disk from other disks.
```

Comment:
```suggestion
When using multiple tablespaces, a best practice is to also add a second hash-partitioned dimension to your hypertable and to have at least one hash partition per disk. While a single time dimension would also work, it would mean that the first chunk is written to one tablespace, the second to another, and so on, and thus would parallelize only if a query's time range exceeds a single chunk.

When adding a hash partitioned dimension, set the number of partitions to a multiple of number of disks. For example, the number of 
partitions P=N*Pd where N is the number of disks and Pd is the number of partitions per 
disk. This enables you to add more disks later and move partitions to the new disk from other disks.
```